requirements.txt

```diff
@@ -1,8 +1,8 @@
 gunicorn>=19.9.0,<19.9.99
 
 # We're waiting on Django 3 until it's further into the lifecycle.
-Django>=2.2.9,<2.2.9.99
-whitenoise>=4.1.4,<4.1.499
+Django>=2.2.10,<2.2.99
```

Comment:
I figured I'd just bump this now. Interesting that in the time I've been working on this, the `jsonfield` sub-dependency had another minor release ('2.1.0 -> 2.1.1`). The `diff` in the description has been updated.